src/components/app-home/app-home.tsx

```diff
@@ -1,15 +1,43 @@
-import { Component, h } from '@stencil/core';
+import { Component, ComponentInterface, h } from '@stencil/core';
 import { getComponents } from '../../utils/component-utils';
 
 @Component({
   tag: 'app-home',
   styleUrl: 'app-home.css'
 })
-export class AppHome {
+export class AppHome implements ComponentInterface {
   components = getComponents();
 
+  private theme: string;
+
+  connectedCallback(): void {
+    const config = JSON.parse(window.sessionStorage.getItem('ionic-persist-config')) ?? {};
+    this.theme = config.theme;
+  }
+
   toggleDarkMode = () => {
-    document.documentElement.classList.toggle('ion-theme-dark');
+    document.documentElement.classList.toggle('ion-palette-dark');
+  }
+
+  /**
+   * Changes the theme to the Ionic theme.
+   * Prompts the user to reload the page for the changes to take effect.
+   */
+  toggleIonicTheme = () => {
+    const config = JSON.parse(window.sessionStorage.getItem('ionic-persist-config')) ?? {};
+
+    if (config.theme === 'ionic') {
+      window.sessionStorage.removeItem('ionic-persist-config');
+    } else {
+      window.sessionStorage.setItem('ionic-persist-config', JSON.stringify({
+        ...config,
+        theme: 'ionic'
+      }));
+    }
+    const confirm = window.confirm('Please reload the page for the changes to take effect.');
```

Comment:
```suggestion
    const confirm = window.confirm('The page will be reloaded to apply your changes.');
```
... or something like that, anyway. The current message sounds like you have to manually reload the page after closing the box, so I got confused when it did it for me.